.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build + Lint
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+      - name: Get yarn cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
```

Comment:
GitHub changed the way you define Outputs quite a while back and now requires you to define values in pre-defined Enviroment variables.

It's now something like this:
```shell
echo "DIR=$(yarn cache dir)" >> $GITHUB_OUTPUT
```

Reply:
thanks, i'll update it to use that

Reply:
You also need to update the output path in the actions/cache step too ;)